fem/gslib.hpp

```diff
@@ -66,43 +102,32 @@ class FindPointsGSLIB
-       @param[out] ref_pos    Reference coordinates of the found point. Ordered
-                              by vdim (XYZ,XYZ,XYZ...).
-                              Note: the gslib reference frame is [-1,1].
-       @param[out] dist       Distance between the sought and the found point
-                              in physical space. */
-   void FindPoints(const Vector &point_pos, Array<unsigned int> &codes,
-                   Array<unsigned int> &proc_ids, Array<unsigned int> &elem_ids,
-                   Vector &ref_pos, Vector &dist);
+       @param[in]  point_pos       Positions to be found. Must by ordered by nodes
+                                   (XXX...,YYY...,ZZZ).
+       @param[out] gsl_codes       Return codes for each point: inside element (0),
+                                   element boundary (1), not found (2).
+       @param[out] gsl_proc        MPI proc ids where the points were found.
+       @param[out] gsl_elem        Element ids where the points were found.
+       @param[out] gsl_mfem_elem   Element ids corresponding to MFEM-mesh
+                                   where the points were found.
+                                   @a gsl_mfem_elem != @a gsl_elem for simplices
+       @param[out] gsl_ref         Reference coordinates of the found point.
+                                   Ordered by vdim (XYZ,XYZ,XYZ...).
+                                   Note: the gslib reference frame is [-1,1].
+       @param[out] gsl_mfem_ref    Reference coordinates @a gsl_ref mapped to [0,1].
+       @param[out] gsl_dist        Distance between the sought and the found point
+                                   in physical space. */
    void FindPoints(const Vector &point_pos);
    /// Setup FindPoints and search positions
    void FindPoints(Mesh &m, const Vector &point_pos, const double bb_t = 0.1,
                    const double newt_tol = 1.0e-12,  const int npt_max = 256);
 
    /** Interpolation of field values at prescribed reference space positions.
-
-       @param[in] codes       Return codes for each point: inside element (0),
-                              element boundary (1), not found (2).
-       @param[in] proc_ids    MPI proc ids where the points were found.
-       @param[in] elem_ids    Element ids where the points were found.
-       @param[in] ref_pos     Reference coordinates of the found point. Ordered
-                              by vdim (XYZ,XYZ,XYZ...).
-                              Note: the gslib reference frame is [-1,1].
        @param[in] field_in    Function values that will be interpolated on the
                               reference positions. Note: it is assumed that
                               @a field_in is in H1 and in the same space as the
                               mesh that was given to Setup().
        @param[out] field_out  Interpolated values. */
-   void Interpolate(Array<unsigned int> &codes, Array<unsigned int> &proc_ids,
```

Comment:
The PR removes / replaces 2 `public` methods. I think they're used by the DRML project, so keep in mind to update that code promptly.